vowpalwabbit/core/src/reductions/ect.cc

```diff
@@ -173,7 +173,6 @@ uint32_t ect_predict(ect& e, single_learner& base, VW::example& ec)
   uint32_t finals_winner = 0;
 
   // Binary final elimination tournament first
-  ec.l.simple = {FLT_MAX};
```

Comment:
This is necessary. Even though we are calling predict, the simple label should still be set since predict paths require a test_only label